### ISSUE TITLE
Cases and deaths: Switching our source from JHU to WHO

### ISSUE THREAD

**Issue by lucasrodes** [owid/covid-19-data#2784]
The Johns Hopkins Coronavirus Resource Center has announced it will stop collecting data on confirmed cases and deaths of COVID-19 on 10 March. You can learn more about their decision from https://github.com/CSSEGISandData/COVID-19/issues/6577 and [their post](https://coronavirus.jhu.edu/from-our-experts/johns-hopkins-to-cease-live-covid-19-data-reporting-on-march-10). 

Following this decision, we will continue to report data on confirmed cases and deaths without interruption by relying instead on [the data made available by the WHO](https://covid19.who.int/data). This change will be made before 10 March, so that users of our data get continuous updates. All tools on our website, and all variable names in our dataset, will remain unchanged.

We want to thank the team at Johns Hopkins University for their invaluable work in creating and maintaining a pivotal dataset to help understand the complexity of the COVID-19 pandemic.

**Comment by beansrowning** [comment]
Are Taiwan, Hong Kong, and Macau being handled separately or dropped entirely from the Cases/Deaths metrics?

**Comment by lucasrodes** [comment]
Hi @beansrowning,
Thanks for your message.

We now rely on the WHO COVID-19 Dashboard (see https://covid19.who.int/). You can read more about this in our post: https://ourworldindata.org/covid-jhu-who.

Important notes:
- The data is shared once a week by the WHO.
- The WHO records the cases and deaths in Taiwan, Hong Kong and Macau under China.
- You can access our latest export (8 March 2023) with data from Johns Hopkins University at https://covid.ourworldindata.org/data/owid-covid-data-old.csv.


Best regards,

**Comment by MikhailKuklin** [comment]
Hello. Do I understand correctly that the next update of https://covid.ourworldindata.org/data/owid-covid-data.csv will be on the 15th of March e.g. one week from the 8th of March?

**Comment by lucasrodes** [comment]
Hi @MikhailKuklin,
This file is updated on a daily basis. However, the metrics concerning confirmed cases and deaths are only updated whenever the WHO updates their dataset. The WHO has announced that they are updating their dataset once a week, on Wednesdays.

In summary, new confirmed cases and deaths values should be available every Thursday at https://covid.ourworldindata.org/data/owid-covid-data.csv

**Comment by MikhailKuklin** [comment]
> Hi @MikhailKuklin, This file is updated on a daily basis. However, the metrics concerning confirmed cases and deaths are only updated whenever the WHO updates their dataset. The WHO has announced that they are updating their dataset once a week, on Wednesdays.
> 
> In summary, new confirmed cases and deaths values should be available every Thursday at https://covid.ourworldindata.org/data/owid-covid-data.csv

Thank you @lucasrodes. If I understand correctly, columns with the metrics concerning confirmed cases and deaths pass `nan` before Thu?

**Comment by lucasrodes** [comment]
@MikhailKuklin Yes! If of interest, we have this dataset with only confirmed cases/deaths: https://github.com/owid/covid-19-data/blob/master/public/data/cases_deaths/full_data.csv. Note that not all metrics are present there and that some names might differ from the primary dataset (https://covid.ourworldindata.org/data/owid-covid-data.csv).

**Comment by jeremyg7** [comment]
it seems like updating USA daily/weekly covid deaths and cases seems to have stopped around May 21 unless i'm missing something?

**Comment by jeremyg7** [comment]
so looking closer to both the full dataset and the cases/deaths dataset, some countries like the USA seem to have stopped updating entirely in mid-May. 

it looks like last reported covid deaths in the USA from WHO is 1,943 on May 14 (both in your dataset and dashboard).

some countries still seem to update (South Korea, Italy, Israel), others (USA, Japan) seem to have just 0's reported for weeks. guessing a WHO issue since it seems to mirror their dashboard, but wondering if any info on what to expect going forward.

**Comment by lucasrodes** [comment]
Hi @jeremyg7,
Thanks for your message.

We publish WHO's data without any alterations, so it looks like an issue for which the WHO will be better positioned to comment.

I just reached out to the WHO asking for more information in this regard.

**Comment by jeremyg7** [comment]
any further info? doesn't look like the WHO has updated USA counts in two months.

**Comment by beansrowning** [comment]
> any further info? doesn't look like the WHO has updated USA counts in two months.

Many countries ceased reporting or changed reporting patterns to WHO following the ending of the PHEIC on 5/5, and/or national emergency declarations. Following the ending of the Public Health Emergency declaration in the US on 5/11, there is no longer a mandatory requirement for states to submit case counts in the way they had while the declaration was in place.

Even prior to that, case reporting in the US and many other countries has been very skewed as most opt for at-home Rapid Antigen Tests versus PCR, which are reportable via the clinical or public health laboratory they're processed in. I suspect for those reasons and many other political ones, case reporting was no longer viable.

Japan, as you identified, also scaled back its national response and categorized COVID-19 as a Category 5 Infectious Disease on 5/9, in line with Influenza. Many of the nationally reported metrics have stopped updating as a result.
